client/reader/sidebar/_style.scss

```diff
@@ -31,55 +34,198 @@
 		}
 	}
 
-	.sidebar-streams {
-	}
+	.sidebar-streams {}
 
-	.sidebar-dynamic-menu__list,
-	.sidebar-dynamic-menu__tag,
-	.sidebar-dynamic-menu__explore {
-		a:first-child {
-			padding: 8px 16px 8px 24px;
+	.is-togglable {
+
+		@include breakpoint( "<660px" ) {
+			margin-top: 0;
 		}
 
-		a.add-new {
-			margin-top: 5px;
+		.sidebar__heading {
+			margin: 0;
+			padding: 10px 16px 10px 55px;
+			cursor: pointer;
+			position: relative;
+			transition: background-color 0.15s ease-in-out,
+				box-shadow 0.15s ease-in-out;
+
+			.gridicon {
+				top: 10px;
+				left: 20px;
+				height: 16px; // I know, I know...
```

Comment:
Well, 16px may be a little smaller than what's normally recommended for a gridicon. "I know, I know..."